app/models/follow.rb

```diff
@@ -0,0 +1,31 @@
+class Follow < ActiveRecord::Base
+  belongs_to :user
+  #TODO Rock&RoR: Check touch usage on cache system
+  belongs_to :followable, polymorphic: true
```

Comment:
I think here the `counter_cache` usage I suggested for the `belongs_to :user` would actually be used, so its more like a performance improvement for any `@investment.follows.count` or `@proposal.follows.count` around. What do you think about naming it "followers_count" twitter-style? 😄